Classes/common/CDTIncrementalStore.m

```diff
@@ -950,10 +962,10 @@ - (BOOL)updateManagedObject:(NSManagedObject *)mo error:(NSError **)error
         }
         if ([prop isKindOfClass:[NSAttributeDescription class]]) {
             NSAttributeDescription *att = prop;
-            enc = [self encodeAttribute:att withValue:changed[name] blobStore:blobStore error:&err];
+            enc = [self encodeAttribute:att withValue:changes[name] blobStore:blobStore error:&err];
         } else if ([prop isKindOfClass:[NSRelationshipDescription class]]) {
```

Comment:
IIRC, Batch updates and relationships do not play well.
I don't think this is something we need to worry or enforce in our code, but I'd like to hear your thoughts on the matter.